service/awsconfig/v10/key/key.go

```diff
@@ -165,6 +165,10 @@ func CustomerID(customObject v1alpha1.AWSConfig) string {
 	return customObject.Spec.Cluster.Customer.ID
 }
 
+func EtcdVolumeName(customObject v1alpha1.AWSConfig) string {
+	return fmt.Sprintf("%s-etcd", ClusterID(customObject))
+}
```

Comment:
Key helper for the volume name. TODO Use this in the Instance Adapter in the CF resource.

Reply:
I'll do this in a later PR as this is already big.